Updating CallRecords API documentation - New fields added (#20859)
microsoftgraph/microsoft-graph-docs: api-reference/beta/resources/callrecords-mediastream.md

|averageAudioDegradation|Double|Average Network Mean Opinion Score degradation for stream. Represents how much the network loss and jitter has impacted the quality of received audio.|
|averageAudioNetworkJitter|Duration|Average jitter for the stream computed as specified in [RFC 3550][], denoted in [ISO 8601][] format. For example, 1 second is denoted as `'PT1S'`, where 'P' is the duration designator, 'T' is the time designator, and 'S' is the second designator.|
|averageBandwidthEstimate|Int64|Average estimated bandwidth available between two endpoints in bits per second.|

> |averageFreezeDuration|Duration|Average for the received freeze duration related with video stream.|

Reviewer: ```suggestion
|averageFreezeDuration|Duration|Average of the received freeze duration related to the video stream.|
```

|averageVideoFrameRate|Double|Average frames per second received for a video stream, computed over the duration of the session.|
|averageVideoPacketLossRate|Double|Average fraction of packets lost, as specified in [RFC 3550][], computed over the duration of the session.|
|endDateTime|DateTimeOffset|UTC time when the stream ended. The DateTimeOffset type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is `2014-01-01T00:00:00Z`|

> |isAudioForwardErrorCorrectionUsed|Boolean|Indicates if the forward error correction (FEC) was used at some point during the session. Null if no information was registered.|

Reviewer: @Datar89 does the property have a default value?

> |isAudioForwardErrorCorrectionUsed|Boolean|True if the forward error correction (FEC) was used at some point during session, false otherwise. Null if no information was registered.|

Reviewer: ```suggestion
|isAudioForwardErrorCorrectionUsed|Boolean|Indicates if the forward error correction (FEC) was used at some point during the session. Null if no information was registered.|
```

> |isAudioForwardErrorCorrectionUsed|Boolean|True if the forward error correction (FEC) was used at some point during session, false otherwise. The default value is null in the case that no information was registered.|

Reviewer: ```suggestion
|isAudioForwardErrorCorrectionUsed|Boolean|Indicates if the forward error correction (FEC) was used at some point during the session. The default value is null.|
```

|maxRoundTripTime|Duration|Maximum network propagation round-trip time computed as specified in [RFC 3550][], denoted in [ISO 8601][] format. For example, 1 second is denoted as `'PT1S'`, where 'P' is the duration designator, 'T' is the time designator, and 'S' is the second designator.|
|packetUtilization|Int64|Packet count for the stream.|
|postForwardErrorCorrectionPacketLossRate|Double|Packet loss rate after FEC has been applied aggregated across all video streams and codecs.|

> |rmsFreezeDuration|Duration|Root mean square for the received freeze duration related with video stream.|

Reviewer 2: It appears `rmsFreezeDuration` may have been missed in the JSON representation below: https://github.com/microsoftgraph/microsoft-graph-docs/blob/13cfa77476bc58d4e2548a07fd2d5a2d2ea033b7/api-reference/beta/resources/callrecords-mediastream.md?plain=1#L91-L92

> |rmsFreezeDuration|Duration|Root mean square for the received freeze duration related with video stream.|

Reviewer: ```suggestion
|rmsFreezeDuration|Duration|Root mean square of the received freeze duration related to the video stream.|
```
